gerrytools/geometry/compactness.py

```diff
@@ -0,0 +1,120 @@
+from gerrychain import (
+    Graph,
+    Partition, 
+    GeographicPartition
+)
+from gerrychain.updaters import (
+    perimeter,
+    exterior_boundaries,
+    interior_boundaries,
+    boundary_nodes,
+    cut_edges_by_part,
+    Tally, # TODO: ask why this isn't already in GeographicPartition
+    perimeter, 
+    exterior_boundaries, 
+    interior_boundaries
+)
+
+from geopandas import GeoDataFrame
+from cv2 import minEnclosingCircle
+from shapely.ops import unary_union
+from .dissolve import dissolve
+from math import pi, sqrt
+import numpy as np
+
+def _reock(partition: Partition, gdf: GeoDataFrame, crs: str):
```

Comment:
As discussed, let's try to eliminate `crs` (at least as a required parameter)—this shouldn't be the responsibility of these scoring functions.